packages/vscode-extension/src/plugins/network/network-plugin.ts

```diff
@@ -1,39 +1,32 @@
 import { commands, Disposable, window } from "vscode";
-import { RadonInspectorBridge } from "../../project/bridge";
+import { NetworkBridge, RadonInspectorBridge } from "../../project/bridge";
 import { ToolKey, ToolPlugin } from "../../project/tools";
 import { extensionContext } from "../../utilities/extensionContext";
-
 import { Logger } from "../../Logger";
 import { NetworkDevtoolsWebviewProvider } from "./NetworkDevtoolsWebviewProvider";
-import { disposeAll } from "../../utilities/disposables";
-import { openContentInEditor, showDismissableError } from "../../utilities/editorOpeners";
-
-import { RequestData, RequestOptions } from "../../network/types/network";
-import {
-  WebviewMessage,
-  CDPMessage,
-  WebviewCommand,
-  IDEMessage,
-} from "../../network/types/panelMessageProtocol";
-
-import { determineLanguage } from "../../network/utils/requestFormatters";
-import { extractTheme } from "../../utilities/themeExtractor";
-
-type BroadcastListener = (message: WebviewMessage) => void;
-
-function formatDataBasedOnLanguage(body: string, language: string): string {
-  if (language === "json") {
-    try {
-      const parsed = JSON.parse(body);
-      return JSON.stringify(parsed, null, 2);
-    } catch {
-      // If JSON parsing fails, return original body
-    }
-  }
-  return body;
-}
+import { WebviewMessage } from "../../network/types/panelMessageProtocol";
+
+import LegacyInspectorStrategy from "./strategies/LegacyInspectorStrategy";
+import NewInspectorStrategy from "./strategies/NewInspectorStrategy";
 
 export const NETWORK_PLUGIN_ID = "network";
+/**
+ * Toggles usage of handling the new network inspector devtools communication.
+ * Disabled by default until new network inspector is integradated into react native.
+ * The instructions to enable the features are in plugins/network/README.md
+ */
+const ENABLE_NEW_INSPECTOR = false;
+
+export type BroadcastListener = (message: WebviewMessage) => void;
+
+export interface InspectorStrategy {
```

Comment:
`Inspector` is an overloaded term in Radon -- there's the "Network Inspector", "Element Inspector", "InspectorBridge" (which you even reference in this file)... I think it'd be better to be explicit in this case, and call this `NetworkInspectorStrategy`